tests/kernel/install_ltp.pm

```diff
@@ -30,22 +30,23 @@ sub scc_we_enabled {
     return wait_serial(qr/Workstation Extension/);
 }
 
-sub add_desktop_productivity_module {
-    if (get_required_var('ARCH') eq 'x86_64' and check_var('DISTRI', 'sle') and sle_version_at_least('15')) {
-        zypper_call('ar http://openqa.suse.de/assets/repo/' . get_required_var('REPO_SLE15_MODULE_DESKTOP_PRODUCTIVITY') . ' DPM', dumb_term => 1);
-        zypper_call('--gpg-auto-import-keys ref',                                                                                  dumb_term => 1);
-    }
-}
-
 sub add_we_repo_if_available {
     # opensuse (doesn't have extensions) or SLE as registered product
     if (check_var('DISTRI', 'opensuse') || scc_we_enabled) {
         return;
     }
+    my $ar_url;
+    my $we_repo = get_var('REPO_SLE_WE15_POOL');
+    if ($we_repo and check_var('DISTRI', 'sle') and sle_version_at_least('15')) {
```

Comment:
`&&` not `and`

Reply:
Why? if && is wanted, why is it not added to the code verification on travis?

Reply:
It has weaker precedence which is different to the `and` operator in most programming languages. It should be disallowed for people's sanity, but someone has to do a code transformation to convert all the `and`s to `&&` first taking the difference in precedence into account.

Reply:
I'm sorry for keeping forgetting about it. I learned using `and` many years ago. I'll be better next time.
@SergioAtSUSE If you create perlformat rule I volunteer to fix the occurrences.